### PR TITLE
[unity] 14611 Unity Location UserUpdate

### PR DESCRIPTION
**Pull request by seaders**
@Sergio-Mira / @adam-govan / @dominicmarmionswrve please

### CODE REVIEW

File: unity3d/Assets/Swrve/SwrveSDK/SwrveNativeIOS.cs [hunk 1]
```diff
@@ -428,12 +431,21 @@ private void showNativeConversation(string conversation)
...
             SwrveLog.LogWarning("Couldn't start Location on iOS correctly, make sure you have the iOS plugin inside your project and you are running on a iOS device: " + exp.ToString());
         }
     }
 
+    public void LocationUserUpdate(Dictionary<string, string> map)
+    {
+        try {
+            _swrveiOSLocationUserUpdate(Json.Serialize(map));
+        } catch (Exception exp) {
+            SwrveLog.LogWarning ("Couldn't update location details from Android: " + exp.ToString ());
```
Comment (Sergio-Mira) [inline unity3d/Assets/Swrve/SwrveSDK/SwrveNativeIOS.cs]: iOS?

File: unity3d/Assets/Swrve/SwrveSDK/SwrveNativeIOS.cs [hunk 2]
```diff
@@ -428,12 +431,21 @@ private void showNativeConversation(string conversation)
     private void startNativeLocation()
     {
         try {
```
Comment (Sergio-Mira) [inline unity3d/Assets/Swrve/SwrveSDK/SwrveNativeIOS.cs:436]: On the Android one we check if we are OnDevice

**Comment by Sergio-Mira** [comment]
🚀

File: unity3d/Assets/Swrve/SwrveSDK/SwrveNativeIOS.cs (hunk 2 above)
Comment (seaders) [reply unity3d/Assets/Swrve/SwrveSDK/SwrveNativeIOS.cs:436]: We don't need to do that on iOS, those functions are just dummied up and return null / false, they don't fail nor give an error message on failing.

**Pull request merged**